main/getting-started/start-a-project.md

```diff
@@ -67,7 +67,10 @@ Cycle](/getting-started/development-cycle.md)
   <tr>
     <td>4</td>
     <td><code>agoric install</code></td>
-    <td>Install JavaScript dependencies, which may take a while.</td>
+    <td>Install JavaScript dependencies, which may take a while.<br>
+        Note: When on a Mac, you must first install 
```

Comment:
Just to verify, is this the step for which Xcode is needed, or was it in the `yarn install` step of the Agoric SDK?